index.html

```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<title>Page Title</title>
+	<link rel="stylesheet" type="text/css" media="screen" href="assets/style.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+		integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+
+<body>
+	<header>
+		<div id="header">
+			<div class="part one">
+				<ul class="list">
+					<li><a href=""><i class="fas fa-bars"></i></a></li>
+					<li><a href=""><i class="fas fa-search"></i></a></li>
+					<li><a href="">SPACE & COSMOS</a></li>
+				</ul>
+			</div>
+			<div class="part two"><img
+					src="https://upload.wikimedia.org/wikipedia/commons/7/77/The_New_York_Times_logo.png"
+					alt="the new york times"></div>
+			<div class="part three">
+				<button class="top-button">LOG IN</button>
+				<button class="top-button">SUBSCRIBE NOW</button>
+			</div>
+		</div>
+	</header>
+
+	<main>
```

Comment:
main is also broken, please clear the floats

Reply:
Realized why it was broken, we cleared the floats so now the main element actually has a size.